src/async-input.js

```diff
@@ -58,6 +58,12 @@ export const AsyncInputAttributes = {
   NAME: 'name',
 };
 
+export const SUBMIT_TIMEOUT_TYPE = Object.freeze({
+  REGULAR: 10000,
+  /* 5 minutes */
```

Comment:
If users really take up to 5 minutes to fill out the payment form, IMO we should just remove the timeout and wait indefinitely until the required action is completed.